### PR TITLE
feat: rules page

### PR DESCRIPTION
**Pull request by krishanu2911**
Added the rule page for the quiz app.
Screenshot (461)

### CODE REVIEW

**Review by kjais1720 — APPROVED** [review]
Looks nice and clean with a good use of utility classes👍

File: pages/quiz-rule.html [hunk 1]
```diff
@@ -4,9 +4,62 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maple-ui.netlify.app/CSS/base.css">
+    <link rel="stylesheet" href="../CSS/quiz-category.css">
+    <link rel="stylesheet" href="../CSS/quiz-rule.css">
+    <link rel="stylesheet" href="../CSS/util.css">
     <title>Document</title>
```
Comment (ttahm3d) [inline pages/quiz-rule.html]: Change the title of the page

**Review by ttahm3d — APPROVED** [review]
Looks good.

File: pages/quiz-rule.html (hunk 1 above)
Comment (krishanu2911) [reply pages/quiz-rule.html]: Thanks Tahir , updated

**Pull request merged**